miniapp/io.hpp

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
-#include <string>
 #include <cstdint>
 #include <iosfwd>
 #include <stdexcept>
+#include <string>
 #include <utility>
+#include <vector>
 
+#include <common_types.hpp>
```

Comment:
We group standard and XMC headers always in two groups with a blank line.
I sorted but leave the blank line to confirm to the rest of the code base